src/routes/GroupSettings/ModeratorsSettingsTab/__snapshots__/ModeratorsSettingsTab.test.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`AddModerator renders correctly when adding with suggestions 1`] = `
 <div
-  data-stylename="styles.add-moderator styles.adding"
+  data-stylename="styles.adding"
```

Comment:
Updates to tests based on Tibet's styling edits (So I assume this is ok)

Reply:
oops yes thanks!